### DISCUSSION TITLE
how to extract one table  spread muti pages

### DISCUSSION THREAD

**Discussion by sunday-will** [jsvine/pdfplumber#530] [Ask for help with specific PDFs]
hello!
**problem:**
[here](https://github.com/jsvine/pdfplumber/files/7448321/xxxx.pdf) is a pdf and tables spread muti pages,Is there a solution to extract them?
![image](https://user-images.githubusercontent.com/42138516/139573673-6ad578ab-0734-4e4c-bc15-10c5ba089955.png)

**code:**
```
    with pdfplumber.open("xxxx.pdf") as pdf:
        page = pdf.pages[0]
        tables = page.extract_tables()
        print(tables)
```

**environment:**

- pdfplumber version: 0.5.28
- Python version: 3.6.0
- OS:  win 7

thanks for any suggestions.

**Comment by jsvine** [comment]
Hi @sunday-will, unfortunately there is no simple method in `pdfplumber` to achieve your goal here. You may be able to write some custom code that identifies page-divided tables and merges them, but the implementation would depend on the specifics of the PDF.

**Reply by sunday-will** [reply]
thanks @jsvine , i think so too,thank you very much again.wish the module more and more powful in future!
